
  Allow preflux to perform an installation to localhost.
  Based on 7bea02b62f7dbd59da0762e8bb6decead200a702 (develop @ 2024-07-30).


--- a/preflux-web/src/main/java/ch/post/it/paisa/preflux/process/inventory/InventoryService.java
+++ b/preflux-web/src/main/java/ch/post/it/paisa/preflux/process/inventory/InventoryService.java
@@ -398,2 +402,10 @@ public class InventoryService implements Cache {
             machine = getEddieMachine(hostname);
+        } else if (Pattern.compile("^localhost$").matcher(hostname).matches()) {
+            machine = new Machine() {
+                @Override public String getHostname() { return hostname; }
+                @Override public String getExpectedRemoteHostname() { return "veddie42"; }
+                @Override public String getSshUsername() { return "isa"; }
+                @Override public boolean needsLegacyDocker() { return false; }
+                @Override public boolean isTeddie() { return true; }
+            };
         } else {



